pkg/sql/conn_executor.go

```diff
@@ -1047,6 +1048,10 @@ type connExecutor struct {
 		// LastActiveQuery contains a reference to the AST of the last
 		// query that ran on this session.
 		LastActiveQuery tree.Statement
+
+		// IdleInSessionTimeout is returned by the AfterFunc call that cancels the
+		// session if the idle time exceeds the idle_in_session_timeout.
+		IdleInSessionTimeout *time.Timer
```

Comment:
nit: if you create an interface (your own struct) here, you can make it have a `Stop()` method that's always available, this way you can avoid `if xxx != nil` all over the place.

Reply:
Done